catalog-rest-service/src/main/java/org/openmetadata/catalog/events/TeamWebhookEventPublisher.java

```diff
@@ -0,0 +1,64 @@
+package org.openmetadata.catalog.events;
+
+import java.util.concurrent.TimeUnit;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.openmetadata.catalog.events.errors.EventPublisherException;
+import org.openmetadata.catalog.jdbi3.CollectionDAO;
+import org.openmetadata.catalog.resources.events.EventResource;
+import org.openmetadata.catalog.slack.SlackRetriableException;
+import org.openmetadata.catalog.slack.TeamsMessage;
+import org.openmetadata.catalog.type.ChangeEvent;
+import org.openmetadata.catalog.type.Webhook;
+import org.openmetadata.catalog.util.ChangeEventParser;
+
+@Slf4j
+public class TeamWebhookEventPublisher extends WebhookPublisher {
+  private final Invocation.Builder target;
+  private final Client client;
+
+  public TeamWebhookEventPublisher(Webhook webhook, CollectionDAO dao) {
+    super(webhook, dao);
+    String slackWebhookURL = webhook.getEndpoint().toString();
+    ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+    clientBuilder.connectTimeout(10, TimeUnit.SECONDS);
+    clientBuilder.readTimeout(12, TimeUnit.SECONDS);
```

Comment:
should that connectionTimeout and readTimeout be configurable?

Reply:
added